quinn/src/streams.rs

```diff
@@ -31,6 +31,7 @@ where
     stream: StreamId,
     is_0rtt: bool,
     finishing: Option<oneshot::Receiver<Option<WriteError>>>,
+    reset_event: Option<oneshot::Receiver<VarInt>>,
```

Comment:
This is also a misnomer.

Reply:
Sorry for the messy PR :/

Reply:
No worries, that's why we have code review.